.circleci/config.yml

```diff
@@ -57,6 +55,17 @@ jobs:
           name: Publish Docker Image to Docker Hub
           command: |
             echo "${DOCKERHUB_PASS}" | docker login -u "${DOCKERHUB_USERNAME}" --password-stdin
+            TAG=${CIRCLE_TAG}
+            D_TAG="$(wget -q https://registry.hub.docker.com/v1/repositories/logdna/logdna-agent/tags -O -  | sed -e 's/[][]//g' -e 's/"//g' -e 's/ //g' | tr '}' '\n'  | awk -F: '{print $3}' | sort -g -r | head -1)"
```

Comment:
You might want to try this method. Reduces the pipes and ignores tags that shouldnt be relevant. It also restricts this to just the v1 in this repository with the MAJOR_VERSION grep.  You'll want to add this up at the top. I'll commment there too.

```D_TAG="$(wget -q https://registry.hub.docker.com/v1/repositories/logdna/logdna-agent/tags -O - | jq -r '.[] | .name' | awk '!/-/ && /\./ { print $1}' | grep ${MAJOR_VERSION} | tail -1)"```

Reply:
On line 13 you would want to add the `MAJOR_VERSION=1`

I also think replacing latest everywhere with a `BASE_TAG=sandbox` would probably be a good idea for simplicity and chnges in the future.